### DISCUSSION TITLE
Disabling/removing 'supported' fields for /ServiceProviderConfig endpoint request

### DISCUSSION THREAD

**Discussion by liodnik** [simpleidserver/SimpleIdServer#329] [Q&A] [answered]
Hi! Is it possible to disable fields like **bulk**, **id** (it's optional) for [SCIM Service Provider Configuration Schema](https://www.rfc-editor.org/rfc/rfc7643#section-5) with the **SimpleIdServer**?

**Comment by simpleidserver** [comment, marked as answer]
Hello,

At the moment, fields like "bulk" or "id" cannot be disabled from the "ServiceProviderConfig" endpoint.
The "id" parameter is optional & it should be possible to enable or disable it.
However, according to the RFC, the "bulk" parameter is required.

I create a ticket (https://github.com/simpleidserver/SimpleIdServer/issues/330) to enable or disable the "id" parameter and set a default value.

**Reply by liodnik** [reply]
Just to clarify -- is it possible to set `"supported" = false` for the "bulk" with _SimpleIdServer_?

**Reply by simpleidserver** [reply]
"Supported" is always true because bulk operation cannot be disabled.

**Reply by liodnik** [reply]
But according to [SCIM Protocol Specification: Bulk Operations](https://www.rfc-editor.org/rfc/rfc7644#section-3.7) the SCIM bulk operation is an optional server feature and I believe it can be `"supported": "false"`.

**Reply by simpleidserver** [reply]
Sorry, my last answer was not very clear.
The "Supported" is always true, because in SimpleIdServer implementation it is not possible possible to disable it.
You're right, to respect the RFC it should be possible to disable it.
A new ticket has been created in order to fix this issue : https://github.com/simpleidserver/SimpleIdServer/issues/332
